### PR TITLE
Make glass and bone knives not conduct electricity and shocks from grille

### PR DESCRIPTION
**Pull request by remuluson2**
## About The Pull Request
Just making shivs more consistent with what is expected. It's weird when glass shard allows you to smash grille connected to power while shivs made out of glass dont. Extended to bone knives for the same reason.
## Why It's Good For The Game
Immersium
## Changelog
:cl: Remuluson2
fix: After pressure from Nanotrasen, Space Wizard Federation made glass and bone knives non-conductive again.
/:cl:

### CODE REVIEW

File: code/game/objects/items/knives.dm [hunk 1]
```diff
@@ -148,6 +148,10 @@
 	throwforce = 15
 	custom_materials = null
 
+/obj/item/knife/combat/bone/Initialize(mapload)
+	flags_1 &= ~CONDUCT_1
+	. = ..()
```
Comment (san7890) [inline code/game/objects/items/knives.dm:153]:
```suggestion
	return ..()
```

this does the exact same thing but is much cleaner to read

File: code/game/objects/items/knives.dm [hunk 2]
```diff
@@ -169,6 +173,11 @@
 	armor_type = /datum/armor/none
 	custom_materials = list(/datum/material/glass=400)
 
+/obj/item/knife/shiv/Initialize(mapload)
+	flags_1 &= ~CONDUCT_1
+	. = ..()
```
Comment (san7890) [inline code/game/objects/items/knives.dm:178]:
```suggestion
	return ..()
```

ditto

File: code/game/objects/items/knives.dm [hunk 3]
```diff
@@ -169,6 +173,11 @@
...
+
+
```
Comment (san7890) [inline code/game/objects/items/knives.dm:180]:
```suggestion
```

no need for two newlines, one is enough

**Review by san7890 — CHANGES_REQUESTED** [review]
style nits

**Comment by remuluson2** [comment]
> style nits

thanks for pointing that out. Not really used to coding in BYOND so I just had no idea return was a thing in this language.

**Pull request merged**